README.md

```diff
@@ -1,24 +1,14 @@
 # README
+git clone this repo
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Use rvm (or the ruby manager of your choice) to install and set your Ruby to the version of Ruby listed in the Gemfile
 
-Things you may want to cover:
+If you have to install a new version of Bundler you’ll need to run gem install bundler
```

Comment:
You can use Markdown to show commands/code:

```suggestion
If you have to install a new version of Bundler you’ll need to run `gem install bundler`
```

This would be good in a few other places too